sprimber-engine/src/main/java/com/griddynamics/qa/sprimber/reporting/TestCaseSummaryPrinter.java

```diff
@@ -96,46 +96,54 @@ public void targetNodeError(SprimberEventPublisher.TargetNodeErrorEvent errorEve
 
     @EventListener
     public void beforeNodeCompleted(SprimberEventPublisher.BeforeNodeCompletedEvent completedEvent) {
-        StringBuilder stringBuilder = reportBuilder.get();
-        stringBuilder.append("\n");
-        stringBuilder.append(getIndents());
-        stringBuilder.append(String.format(" %s", completedEvent.getNode().getName()));
-        stringBuilder.append(String.format(" (%s) ", completedEvent.getNode().getCurrentState()));
-        printMethodParameters(completedEvent.getNode().getMethodParameters());
+        if ("Before Test".equals(completedEvent.getNode().getRole()) || "Before Step".equals(completedEvent.getNode().getRole())) {
```

Comment:
Looks like some workaround. I also don't like usage of magic strings here

Reply:
magic strings will gone in the next review